dashboard/modules/job/data_types.py

```diff
@@ -31,6 +32,8 @@ class JobSubmitRequest:
     runtime_env: Dict[str, Any]
     # Command to start execution, ex: "python script.py"
     entrypoint: str
+    # Optional job_id to specify for the job.
```

Comment:
Maybe add a comment documenting the semantics (what happens when job_id is present/not present)

Reply:
yeah good idea